app/sequencescape_excel/sequencescape_excel/download.rb

```diff
@@ -1,36 +0,0 @@
-# frozen_string_literal: true
```

Comment:
This file wasn't needed, as we don't inherit from it. So I've removed it to avoid confusion.

Reply:
When I added spreadsheet generation for bulk submission, I copied a lot of the behaviour here, and originally intended to inherit from this. But this never really happened for downloads, so this file just sat there orphaned and I didn't clean it up at the time.